core/src/sql/value/serde/de/mod.rs

```diff
@@ -174,6 +173,8 @@ fn into_json(value: Value, simplify: bool) -> JsonValue {
 		}
 	}
 
+	// TODO: Checkout blame for why these are here.
```

Comment:
Still not sure why this was here, it seems to be unused but I can't find the change which made it unused.